std/range/package.d

```diff
@@ -2836,8 +2836,14 @@ Cycle!R cycle(R)(R input)
 @safe unittest
 {
     import std.algorithm : equal;
-
-    assert(equal(take(cycle([1, 2][]), 5), [ 1, 2, 1, 2, 1 ][]));
+    import std.range : cycle, take;
+
+    // Here we create an infinitive cyclic sequence from [1, 2]
+    // (i.e. get here [1, 2, 1, 2, 1, 2 and so on]) then
+    // take 5 elements of this sequence (so we have [1, 2, 1, 2, 1]) 
+    // and compare it with specific values - they should
+    // be equal of course.
```

Comment:
"they should be equal of course" sound to informal IMO.

what about, "... and compare them with the expected values for equality" ?